From: Feng Kan <fkan@apm.com>
Date: Wed, 4 Dec 2013 13:33:32 -0500
Subject: [PATCH 1/2] arm64: add missing dts entry for X-Gene platform.
Origin: http://lists.infradead.org/pipermail/linux-arm-kernel/2013-December/216347.html

This add missing dts entry for X-Gene reboot driver.

Signed-off-by: Feng Kan <fkan at apm.com>

[ijc -- obsoleted by upstream syscon-reboot driver reworking, but that is too
 big to backport.]
[bwh: Forward-ported to 3.19. Upstream now has both drivers and still nothing
 in this file.]
---
 arch/arm64/boot/dts/apm/apm-storm.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/boot/dts/apm/apm-storm.dtsi
+++ b/arch/arm64/boot/dts/apm/apm-storm.dtsi
@@ -480,6 +480,11 @@
 			interrupts = <0x0 0x4c 0x4>;
 		};
 
+		reboot@17000014 {
+			compatible = "apm,xgene-reboot";
+			reg = <0x0 0x17000014 0x0 0x4>;
+		};
+
 		serial1: serial@1c021000 {
 			status = "disabled";
 			device_type = "serial";
